docs/2.6.3.md

```diff
@@ -0,0 +1,48 @@
+# ProxySQL 2.6.3 and ProxySQL admin tools (2024-06-04)
+
+[Download](https://www.percona.com/download-proxysql)<br>
+[Install](install-v2.md)<br>
+[View the GitHub repository](https://github.com/percona/proxysql-admin-tool)
+
+[ProxySQL](https://proxysql.com/) is a high-performance proxy for MySQL and MySQL-compatible database servers such as Percona Server for MySQL and MariaDB. It acts as an intermediary for client requests seeking resources from the database. ProxySQL was created for the database administrator to solve complex replication topology issues.
+
+You can [download the original ProxySQL from GitHub](https://github.com/sysown/proxysql/releases). The [ProxySQL documentation](https://proxysql.com/documentation/) provides information on installing and running ProxySQL.
+
+The [ProxySQL Admin (proxysql-admin)](proxysql-admin-tool-v2-config.md) tool simplifies the configuration of Percona XtraDB Cluster nodes with ProxySQL. [ProxySQL Admin 1](proxysql-v1.md) requires custom `Bash` scripts to track the status of a Percona XtraDB Cluster. [ProxySQL Admin 2](proxysql-admin-tool-functions.md) supports Percona XtraDB Cluster without custom scripts.
+
+The [ProxySQL 2 and pxc_scheduler_handler](psh-overview.md) tool can automatically perform a failover due to node failures, service degradation, or maintenance. The pxc_scheduler_handler tool has different features and capabilities than ProxySQL admin. Do not use the options from one tool in the other tool. Mixing the options may cause unintended results.
+
+## Release highlights
+
+Compared to the previous [ProxySQL 2.6.2 release], this release includes all improvements and bug fixes available in [ProxySQL 2.6.3].
+
+* Added a hostgroup attribute, `monitor_slave_lag_when_null`.
+
+* The system now prioritizes over the global variable `mysql-monitor_slave_lag_when_null`.
+
+* Negative values for `Seconds_Behind_Master` bring previously shunned servers back online.
+
+* The table `mysql_server_replication_lag_log` now logs negative values for `Seconds_Behind_Master`.
+
+* A new safety feature is introduced via the hostgroup attribute `max_num_online_servers`.
+
+* The system disables routing traffic to the hostgroup when the attribute value is exceeded.
+
+## Bug fixes
+
+* [PSQLADM-514]: The `syncusers` user account does not need write permission to proxysql dir.
```

Comment:
We decided to not fix `PSQLADM-514`. Please remove this from the release notes.

Reply:
Done